.github/workflows/rust.yml

```diff
@@ -49,7 +49,16 @@ jobs:
       - name: Set PKG_CONFIG_PATH
         run: echo "PKG_CONFIG_PATH=/usr/lib/x86_64-linux-gnu/pkgconfig" >> $GITHUB_ENV
       - name: Run clippy
-        run: cargo clippy --all-targets --all-features -- -D warnings
+        run: |
+          cargo clippy --all-targets --features \
```

Comment:
so with this, we are now missing features. The original was running with:
```
jemalloc = ["librocksdb-sys/jemalloc"]
io-uring = ["librocksdb-sys/io-uring"]
valgrind = []
mt_static = ["librocksdb-sys/mt_static"]
snappy = ["librocksdb-sys/snappy"]
lz4 = ["librocksdb-sys/lz4"]
zstd = ["librocksdb-sys/zstd"]
zlib = ["librocksdb-sys/zlib"]
bzip2 = ["librocksdb-sys/bzip2"]
rtti = ["librocksdb-sys/rtti"]
multi-threaded-cf = []
serde1 = ["serde"]
bindgen-runtime = ["librocksdb-sys/bindgen-runtime"]
```

can we update this to match? We basically want to run all features minus running bindgen-runtime & bindgen-static since they conflict with each other right?

Reply:
The default features are enabled by default.

Reply:
Maybe you mean explicitly listing them is better?

Reply:
> The default features are enabled by default. I think?

you are right, I thought we are only running the features being explicitly listed. This looks good to me.